src/bindings/python/src/pyopenvino/frontend/pytorch/extension.cpp

```diff
@@ -0,0 +1,78 @@
+// Copyright (C) 2023 Intel Corporation
+// SPDX-License-Identifier: Apache-2.0
+//
+
+#include "extension.hpp"
+#include "utils.hpp"
+
+#include <pybind11/functional.h>
+#include <pybind11/pybind11.h>
+#include <pybind11/stl.h>
+#include <pybind11/stl_bind.h>
+
+#include "openvino/frontend/extension/conversion.hpp"
+#include "openvino/frontend/pytorch/extension/conversion.hpp"
+#include "openvino/frontend/pytorch/extension/op.hpp"
+
+namespace py = pybind11;
+
+using namespace ov::frontend::pytorch;
+
+
+void regclass_frontend_pytorch_ConversionExtension(py::module m) {
+    py::class_<ConversionExtension, ConversionExtension::Ptr, ov::frontend::ConversionExtensionBase> _ext(
+        m,
+        "_ConversionExtensionPytorch",
+        py::dynamic_attr());
+    class PyConversionExtension : public ConversionExtension {
+    public:
+        using Ptr = std::shared_ptr<PyConversionExtension>;
+        using PyCreatorFunction = std::function<ov::OutputVector(const ov::frontend::NodeContext*)>;
+        PyConversionExtension(const std::string& op_type, const PyCreatorFunction& f)
+            : ConversionExtension(op_type, [f](const ov::frontend::NodeContext& node) -> ov::OutputVector {
+                  return f(static_cast<const ov::frontend::NodeContext*>(&node));
+              }) {}
+    };
+    py::class_<PyConversionExtension, PyConversionExtension::Ptr, ConversionExtension> ext(
+        m,
+        "ConversionExtensionPytorch",
+        py::dynamic_attr());
+
+    ext.def(py::init([](const std::string& op_type, const PyConversionExtension::PyCreatorFunction& f) {
+        return std::make_shared<PyConversionExtension>(op_type, f);
+    }));
+}
+
+void regclass_frontend_pytorch_OpExtension(py::module m) {
+    py::class_<OpExtension<void>, std::shared_ptr<OpExtension<void>>, ConversionExtension> ext(
+            m,
+            "OpExtensionPytorch",
+            py::dynamic_attr());
+
+    ext.def(py::init([](const std::string& fw_type_name,
+                        const std::map<std::string, size_t>& attr_names_map,
+                        const std::map<std::string, py::object>& attr_values_map) {
+                std::map<std::string, ov::Any> any_map;
+                for (const auto& it : attr_values_map) {
+                    any_map[it.first] = Common::utils::py_object_to_any(it.second);
```

Comment:
Is it correct to throw exception in the ctor in python?
py_object_to_any can throw exception in case of the py_object can not be representes as c++ type